src/handle.rs

```diff
@@ -111,6 +112,12 @@ impl<'s, T> Deref for Local<'s, T> {
   }
 }
 
+impl<'s, T> DerefMut for Local<'s, T> {
+  fn deref_mut(&mut self) -> &mut T {
+    unsafe { self.0.as_mut() }
+  }
+}
```

Comment:
As per @piscisaureus's comment we should remove this

Reply:
`instance_of` requires for `T` to be mutable in `v8::Local<T>`, hence the deref_mut implementation. I'm not sure how you can do `instance_of` on a local handle without this.

Reply:
Is there a reason the `InstanceOf` function is not marked as `const` in the c++ api?
Seems counter-intuitive, could this be a mistake in the v8 c++ api?

Reply:
@inteon See https://github.com/denoland/rusty_v8/pull/879#discussion_r790182815